next/13/replace-next-head-v2/test.ts

```diff
@@ -360,21 +359,21 @@ describe('next 13 replace-next-head-v2', function () {
 		const A_CONTENT = `
 		import Meta from '../../components/a.tsx';
 		
-		export default function Page({ title, description, appName }) {
-			return <Meta title={title} description={description} appName={appName} />;
+		export default function Page({ title, description }) {
```

Comment:
what's the reason for these changes?

Reply:
i wanted to test the case when one of props is passed as StringLiteral, so i've removed appName from parent and assigned it to some value in child component.

Reply:
`<NestedComponent appName={"appName"} />`